gpkit/nomials/array.py

```diff
@@ -116,18 +115,6 @@ def __bool__(self):
         "Allows the use of NomialArrays as truth elements in python3."
         return all(bool(p) for p in self.flat)
 
-    @property
-    def c(self):
```

Comment:
This was a weird idea that we never used, so I figured it should be cleaned out.